presidio-analyzer/analyzer/nlp_engine/spacy_nlp_engine.py

```diff
@@ -14,6 +15,7 @@ class SpacyNlpEngine(NlpEngine):
 
     def __init__(self):
         logger.info("Loading NLP model...")
+        download('en_core_web_lg')
```

Comment:
does this means that if the file already exists it will be replaced? or we need to delete it from the repo?

Reply:
it only downloads in case the model isn't downloaded already. Otherwise it does nothing

Reply:
This is important to document, as a comment

Reply:
@itye-msft added.